anisoap/reference/projection_coefficients.py

```diff
@@ -1,11 +1,9 @@
 import numpy as np
 
 from ..representations.radial_basis import RadialBasis
+from ..utils import quaternion_to_rotation_matrix  # missing?
```

Comment:
Is this really missing?

Reply:
I'm fine with removing it if it's no longer needed. It might be a relic from the first version.

Reply:
*removing this line

Reply:
yep, we can remove this line in that case